doc/quickstart/find_data.rst

```diff
@@ -363,6 +363,32 @@ Key           Description                   Default value if not specified
               file                          default DRS is used)
 ============= ============================= =================================
 
+ESMValTool automatically makes native ICON data `UGRID
```

Comment:
```suggestion
ESMValCore automatically makes native ICON data `UGRID
```

Reply:
It would be good to talk about ESMValCore when something is done by ESMValCore. This will help users to distinguish the two packages and understand how they interact.